### PR TITLE
refactoring: site header

### PR DESCRIPTION
**Pull request by neSpecc**
- The [@codexteam/nuxt-icons](https://github.com/codex-team/icons/tree/master/packages/nuxt) module added to use CodeX Icons
- ~~[nuxt-icons](https://github.com/gitFoxCode/nuxt-icons) added for working with SVG as `<svg>` instead of ``~~
- [nuxt-svgo](https://github.com/cpsoinos/nuxt-svgo) added for working with SVG as `<svg>` instead of ``
- Markup refactored
- Support Editor.js button redesigned
- Header became sticky
- Header Mobile: menu toggling added

image

image

### CODE REVIEW

File: assets/styles/vars.pcss [hunk 1]
```diff
@@ -1,5 +1,28 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (TatianaFomina) [inline assets/styles/vars.pcss:22]: ?

Comment (neSpecc) [reply assets/styles/vars.pcss:22]: above: refactored, below: to be refactored

Comment (TatianaFomina) [reply assets/styles/vars.pcss:22]: 🫤

**Comment by TatianaFomina** [comment]
`yarn dev` fails
image

**Comment by neSpecc** [comment]
> `yarn dev` fails
> 
> image

It is a bug of `nuxt-icons`, see https://github.com/gitFoxCode/nuxt-icons/pull/26

Won't resolve it for now. If it won't be fixed, I'll get rid of that lib

File: assets/styles/vars.pcss (hunk 1 above)
Comment (neSpecc) [reply assets/styles/vars.pcss]: I've removed empty lines

**Comment by neSpecc** [comment]
> `yarn dev` fails

Fixed. Removed `nuxt-icons`.

**Review by TatianaFomina — APPROVED** [review]

**Pull request merged**